### ISSUE TITLE
Re-export `csjs`?

### ISSUE THREAD

**Issue by tizmagik** [tizmagik/react-csjs#6]
One thing that may help clean up the API surface of using CSJS is to re-export `csjs`. Currently, users need to import both `csjs` and `react-csjs` for any styled components:

``` js
import csjs from 'csjs';
import withStyles from 'react-csjs';
```

Instead, we could have an interface like this:

``` js
import csjs, { withStyles } from 'react-csjs';
```

And in order to not be a bottleneck on `csjs` updates, we can define it as a `peerDependency`.

**Comment by drcmda** [comment]
Here's a way it could work without re-export: https://github.com/tizmagik/react-csjs/pull/9

About the peerDependency, i think since npm 3 they're not fetched automatically, but will return a warning if unresolved. Is this wanted? As long as the readme says 

```
npm install --save csjs react-csjs
```

that would be fine. (Right now it just mentions react-csjs)

**Comment by wKovacs64** [comment]
This can be closed now that #9 has landed, right?

**Comment by tizmagik** [comment]
Ah, yeap! Thanks
